test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -435,9 +435,9 @@ TEST(mongoUpdateContext_withOnchangeSubscriptions, Cond1_updateMatch)
     EXPECT_EQ(SccOk, ms);
 
     /* Check lastNotification */
```

Comment:
I see that 35 test were modified in order to fit to the new way. However, maybe to have a sample of them (e.g. 5 tests?) with cache disabled (it should be a matter of adjusting a global/library variable) is a good idea, as a "watchdog" to ensure that old functionality still working.